deploy/crds.yaml

```
@@ -211,14 +224,10 @@ spec:
           metadata:
             type: object
           spec:
-            description: |-
-              PolicySpec is the spec of the Policy resource.
-              The spec includes multiple fields, where each field represents a different policy.
-              Only one policy (field) is allowed.
+            description: spec field of the Policy resource
```

Older description was better, this is too generic
```suggestion
            description: |-
              PolicySpec is the spec of the Policy resource.
              The spec includes multiple fields, where each field represents a different policy.
              Only one policy (field) is allowed.
```